src/libPMacc/include/particles/memory/dataTypes/Particle.hpp

```diff
@@ -110,6 +111,12 @@ struct Particle : public InheritLinearly<typename T_FrameType::MethodsList>
     >::type
     operator[](const T_Key key)
     {
+        PMACC_CASSERT_MSG_TYPE(
+            key_not_exists_in_particle,
```

Comment:
Please don't use Yoda grammar: `key_does_not_exist`
`_in_particle` could be omitted as the error message would already be in Particle::operator[]